src/components/heading/definition.js

```diff
@@ -1,38 +1,45 @@
 import Heading from './';
 import Definition from './../../../demo/utils/definition';
 
-let definition = new Definition('heading', Heading, {
-  description: `The titles of a page and its sections.`,
+const definition = new Definition('heading', Heading, {
+  description: 'The titles of a page and its sections.',
   designerNotes: `
-* A standard page or section header, with a range of options, such as a help link presented as an icon, a sub-heading, a ‘Back’ icon, and visual dividers and separators.
-* Make sure that the colour of all text has a contrast ratio of at least 4.5.1, to meet the [WCAG 2.0 AA standard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) for text less than 19px in size. Webaim have a good online [Contrast Checker](http://webaim.org/resources/contrastchecker/).
+* A standard page or section header, with a range of options, such as a help link presented as an icon, a sub-heading, a
+ ‘Back’ icon, and visual dividers and separators.
+* Make sure that the colour of all text has a contrast ratio of at least 4.5.1, to meet the [WCAG 2.0 AA standard]
+(https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) for text less than 19px in size. Webaim
+ have a good online [Contrast Checker](http://webaim.org/resources/contrastchecker/).
   `,
   relatedComponentsNotes: `
 * Simple text content? [Try Content](/components/content).
 * Prominent text callout? [Try Detail](/components/detail).
  `,
   type: 'layout',
   propTypes: {
-    children: "Node",
-    className: "String",
-    title: "String",
-    subheader: "String",
-    help: "String",
-    helpLink: "String",
-    backLink: "String",
-    divider: "Boolean",
-    separator: "Boolean"
+    children: 'Node',
+    className: 'String',
+    title: 'String',
+    titleId: 'String',
+    subheader: 'String',
+    subtitleId: 'String',
+    help: 'String',
+    helpLink: 'String',
+    backLink: 'String, Func',
```

Comment:
I would switch this back to a String so it doesn't error in the demo site. This is not ideal but you could add something to the description to say that it can be a function too